src/py42/clients/audit_logs.py

```diff
@@ -0,0 +1,41 @@
+class AuditLogsClient(object):
+    def __init__(self, audit_log_service):
+        self._audit_log_service = audit_log_service
+
+    def get_all(
+        self,
+        begin_time=None,
+        end_time=None,
+        event_types=None,
+        user_ids=None,
+        user_names=None,
```

Comment:
`usernames` if we want to be consistent with other parameters such as `cloud_usernames`.

Reply:
renamed.